.github/workflows/docs.yml

```
@@ -0,0 +1,46 @@
+name: deploy-book
+
+on:
+  push:
+    branches:
+    - main
```

The `-` should be indented with two spaces after branches: to ensure correct YAML structure.

```suggestion
      - main
```